src/routes/api/users/userController.js

```diff
@@ -117,7 +117,8 @@ class Users {
             attributes: { exclude: ['password'] }
           },
           { model: ratings }
-        ]
+        ],
+        order: [['createdAt', 'DESC']]
```

Comment:
Any reason for having a nested array here?

Reply:
Yes, that it is the way Sequelize implements ordering.